websites/M/MyTF1/presence.ts

```diff
@@ -0,0 +1,164 @@
+const presence = new Presence({
+		clientId: "1043951049800695808",
+	}),
+	strings = presence.getStrings({
+		viewHome: "general.viewHome",
+		viewPage: "general.viewPage",
+		viewProfile: "general.viewProfile",
+		watchingVid: "general.watchingVid",
+		browse: "general.browsing",
+		live: "general.live",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	MyTF1 = "https://i.imgur.com/k8jIoSZ.png",
+	TF1 = "https://i.imgur.com/pFgnoUS.png",
+	TFX = "https://i.imgur.com/u5dTkHx.png",
+	TMC = "https://i.imgur.com/62uYw2r.png",
+	LCI = "https://i.imgur.com/qrRgNgk.png",
+	TF1Series = "https://i.imgur.com/WOmFIm3.png",
+	Stream = "https://i.imgur.com/xuvR04J.png",
+	Replay = "https://i.imgur.com/OZDAx9n.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.MyTF1,
+			startTimestamp: browsingTimestamp,
+		},
+		path = document.location.pathname,
+		time = await presence.getSetting<boolean>("time");
+
+	if (!time) delete presenceData.startTimestamp;
+
+	const { browse, live, viewHome, viewProfile, watchingVid } = await strings;
+
+	if (path === "/") {
+		presenceData.details = viewHome;
+		presenceData.state = browse;
+	} else if (path.startsWith("/tf1/direct")) {
+		presenceData.details = document.querySelector(
+			"[class*=VideoSummary_programName]"
+		).textContent;
+		presenceData.state = document.querySelector(
+			"[class*=VideoSummary_title]"
+		).textContent;
+		presenceData.smallImageKey = Assets.Stream;
```

Comment:
Use standardized assets.
```suggestion
		presenceData.smallImageKey = Assets.Live;
```